code/modules/research/techweb/all_nodes.dm

```diff
@@ -1482,6 +1482,7 @@
 		"portaseeder",
 		"seed_extractor",
 		"adv_watering_can",
+		"plantgene",
```

Comment:
Go look at ``modular_zubbers\code\modules\research\techweb\all_nodes.dm`` and remove this.

Reply:
Also in the future make sure you document Bubberstation edits with a comment